docs/reference/commandline/dockerd.md

```diff
@@ -1222,6 +1251,9 @@ This is a full example of the allowed configuration options on Linux:
   "fixed-cidr-v6": "",
   "group": "",
   "hosts": [],
+  "http-proxy": "http://proxy.example.com:80",
+  "https-proxy": "https://proxy.example.com:443",
```

Comment:
this seems to always be a contentious point: the type of HTTP request to use for the proxy, and whether to use use different protocols/ports for HTTP_PROXY and HTTPS_PROXY. We should probably update the [proxy configuration page](https://docs.docker.com/network/proxy/) in a follow-up, to use the same example values, so we're consistent across the board.